streams/src/main/java/org/apache/kafka/streams/kstream/KStream.java

```diff
@@ -53,7 +53,6 @@
  * @see KGroupedStream
  * @see KStreamBuilder#stream(String...)
  */
-@SuppressWarnings("unused")
```

Comment:
This is used as some of the apis are never used within the library itself, curious to learn what's the motivation of removing it.